zebra/rule_netlink.c

```diff
@@ -116,10 +116,11 @@ static int netlink_rule_update(int cmd, struct zebra_pbr_rule *rule)
 
 	if (IS_ZEBRA_DEBUG_KERNEL)
 		zlog_debug(
-			"Tx %s family %s IF %s(%u) Pref %u Src %s Dst %s Table %u",
+			"Tx %s family %s IF %s(%u) Pref %u Fwmark %u Src %s Dst %s Table %u",
```

Comment:
The code comment does not match what this patch does.  Is this intentional?

Reply:
actually, there is nothing to do.
I will reword the commit so as to match with the change.

Reply:
better than this. I remove that commit, and make a separate pull request.